pom.xml

```diff
@@ -140,17 +141,23 @@
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>emitter</artifactId>
-                <version>0.4.1</version>
+                <version>0.4.5</version>
             </dependency>
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>http-client</artifactId>
-                <version>1.0.6</version>
+                <version>1.1.0</version>
             </dependency>
             <dependency>
                 <groupId>com.metamx</groupId>
                 <artifactId>server-metrics</artifactId>
                 <version>0.2.8</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.metamx</groupId>
+                        <artifactId>java-util</artifactId>
```

Comment:
We can't exclude this though. `server-metrics`uses the `metamx` origin of `java-utl`, things in druid.io only use the `io.druid` fork